troposphere/firehose.py

```diff
@@ -60,6 +60,26 @@ class CopyCommand(AWSProperty):
         'DataTableName': (basestring, True),
     }
 
+class ProcessorParameter(AWSProperty):
+    props = {
+        'ParameterName': (basestring, True),
+        'ParameterValue': (basestring, True),
+    }
+
+
+class Processor(AWSProperty):
+    props = {
+        'Parameters': ([ProcessorParameter], True),
+        'Type': (basestring, True),
```

Comment:
it might be worth a custom validator here, since the only valid type it appears is `Lambda`:

http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-kinesisfirehose-deliverystream-processor.html

Reply:
I went ahead and added validator functions for all of the string type enums that I noticed in the docs files for these APIs.